frontend/src/reducers/observation/index.js

```diff
@@ -1,5 +1,10 @@
+import { mapToFrontEnd, mapToBackEnd } from './mapper'
```

Comment:
create a test for this reducer. Coupled with tests for the mappers this will increase test coverage